server/packetio.go

```diff
@@ -56,10 +56,11 @@ type packetIO struct {
 	sequence uint8
 }
 
-func newPacketIO(conn net.Conn) *packetIO {
+func newPacketIO(conn net.Conn, sequence uint8) *packetIO {
 	p := &packetIO{
-		rb: bufio.NewReaderSize(conn, defaultReaderSize),
-		wb: bufio.NewWriterSize(conn, defaultWriterSize),
+		rb:       bufio.NewReaderSize(conn, defaultReaderSize),
+		wb:       bufio.NewWriterSize(conn, defaultWriterSize),
+		sequence: sequence,
```

Comment:
@shenli @tiancaiamao Confirmed load data is working. In fact it wouldn't affect load data because:
- During load data, packets are sent from client to server. The `sequence` here however is the server-to-client packet sequence.
- SSL handshake happens so early that even before authentication.